Frontend/src/locales/en.json

```diff
@@ -76,9 +76,21 @@
   },
   "photos":  {
     "myPhotos": "My Photos",
-    "friendsPhotos": "Photos your friends took"
+    "descriptionMyPhotos": "Here you can view all the photos you've uploaded for the Photowalk challenges. Photos your Friends like the best have the most heart icons.",
+    "friendsPhotos": "Photos my friends took",
+    "descriptionFriendsPhotos": "Here you can view all Photos your Friends took on their Photowalks. Do you recognize any of the motifs? Show your friends you like a Photo they took by klicking on the heart icon!"
   },
   "photowalks": {
-    "chooseWalk": "Choose a Photowalk"
+    "chooseWalk": "Choose a Photowalk",
+    "startWalk": "start Walk"
+  },
+  "mobileView": {
+    "mapTab": "Map",
+    "challengesTab": "Challenges & Info",
+    "photosTab": "Photos"
+  },
+  "landing": {
+    "headline":"Want to take a walk?",
+    "pitch": "Are you in the mood for some fresh air, but think going on a walk on you own is boring? Do you miss doing photo rallies with your friends and comparing your cool photos? PhotoWalk allows you to do just that, regardless of social distancing! Go on one of three curated walks in the most beautiful city in the north. Take your camera, take pictures of some great motives, and compete with your friends in photo challenges. How do you and your friends interpret our challenges? What new things can you discover on your walk? Sign up and find out."
```

Comment:
Are you in the mood for some fresh air, but think going on a walk on your own is boring? Do you miss doing photo rallies with your friends and comparing your cool photos? PhotoWalk allows you to do just that, regardless of social distancing! Go on one of three curated walks in the most beautiful city in the north. Take your camera, take pictures of some great motifs, and compete with your friends in photo challenges. How do you and your friends interpret our challenges? What new things can you discover on your walk? Sign up and find out.